readthedocs/projects/tasks/search.py

```diff
@@ -166,10 +166,9 @@ def _get_indexers(*, version: Version, build: Build, search_index_name=None):
 
     # File tree diff is under a feature flag for now,
     # and we only allow to compare PR previews against the latest version.
-    has_feature = version.project.has_feature(
-        Feature.GENERATE_MANIFEST_FOR_FILE_TREE_DIFF
+    create_manifest = version.project.addons.filetreediff_enabled and (
+        version.is_external or version.slug == LATEST
```

Comment:
Looks like test are failing because projects don't have an addons object. I think we should create one for each project on a post save signal, otherwise we will see that error in production as well.

Reply:
I implemented this, together with a small clean up, in https://github.com/readthedocs/readthedocs.org/pull/11799